trec-car-tools-java dependency Error

Like #1249 and #1025 

Hi, it seems the url of trec-car-tools-java is broken, so I can't download it, how do I fix it? ( maybe a jar file could solve it?)

@lintool @MXueguang 

Thanks!

I can confirm the issue for version 13

[ERROR] Failed to execute goal on project treccar-tools-example: Could not resolve dependencies for project edu.unh.cs.treccar:treccar-tools-example:jar:0.1: Could not find artifact com.github.TREMA-UNH:trec-car-tools-java:jar:13 in jitpack.io (https://jitpack.io) -> [Help 1]
[ERROR] 

and 15
[ERROR] Failed to execute goal on project treccar-tools-example: Could not resolve dependencies for project edu.unh.cs.treccar:treccar-tools-example:jar:0.1: Failure to find com.github.TREMA-UNH:trec-car-tools-java:jar:15 in https://jitpack.io was cached in the local repository, resolution will not be reattempted until the update interval of jitpack.io has elapsed or updates are forced -> [Help 1]


When I switch to version 16, ... 19, it works

[INFO] BUILD SUCCESS


I am appalled if jitpack is unable to serve older versions --- why would we be using those, if they grow defunct?

--- a/pom.xml
+++ b/pom.xml
@@ -414,7 +414,7 @@
     <dependency>
       <groupId>com.github.TREMA-UNH</groupId>
       <artifactId>trec-car-tools-java</artifactId>
-      <version>13</version>
+      <version>19</version>
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>

